src/PlexRequests.TheMovieDb/TheMovieDbApi.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using PlexRequests.Api;
+using PlexRequests.Settings;
+using PlexRequests.TheMovieDb.Models;
+
+namespace PlexRequests.TheMovieDb
+{
+    public class TheMovieDbApi : ITheMovieDbApi
+    {
+        private const string BaseUri = "https://api.themoviedb.org/3/";
```

Comment:
I recommend adding the uri to the appsettings

Reply:
Good point, making the change.